### ISSUE TITLE
Start: The Book/Article/Content Reader

### ISSUE THREAD

**Issue by p-raj** [p-raj/reads#18]
## References
<img width="400" alt="screen shot 2018-09-15 at 11 08 27 pm" src="https://user-images.githubusercontent.com/40564263/46065570-08aff200-c190-11e8-9152-63f949219c5b.png">
<img width="400" alt="screen shot 2018-09-17 at 3 12 02 pm" src="https://user-images.githubusercontent.com/40564263/46065571-09488880-c190-11e8-9813-05845a3fe914.png">
<img width="400" alt="screen shot 2018-09-17 at 3 12 09 pm" src="https://user-images.githubusercontent.com/40564263/46065572-09488880-c190-11e8-8212-c216d45bb29e.png">

# What is it?

Get a summary for the topic of your interest, explore the content from multiple dimensions - focus first on the time.

**Todo**

- [ ] Define problem statement & solution
- [ ] Define market & similar competition
- [ ] Define product & probably the UI
  - [ ] MVP
  - [ ] Long term
- [ ] Target audience
- [ ] Fill in business canvas
- [ ] Pitch deck
  - Introduction
  - Team <if you have a great team - 2nd slide otherwise 2nd last slide>
  - Problem Statement
  - Solution
  - Market Size
      -Target Market
  - KPIs <if there are any>
  - Business Model Opportunities
  - Competition
  - GTM
  - Vision & Mission
  - Roadmap

**Comment by p-raj** [comment]
**TL;DR** – Possible, but have to be categorized topic-wise and per topic there have to be multiple databases with a primary focus on time to go through the content.

### How this would work – for example: Topic: Books. Possible Databases (sources)
- eBooks – Full Length, slowest source
- Audio Books – Full Length, but faster than reading a book
- Youtube Interviews (author’s speeches, Q&A with author) – specific topics and
chaperts from the book
- Twitter threads & #<topic> - any length of threads could be expected
- Blinkist – 15 minutes for a non-fictional book
- User Annotations – eg: genius.com, hypothes.is – most-highlighted line

### MVP (minimal viable product) – Only data source – Twitter Threads Information in twitter threads
- Length of threads
- Words per thread (max 240)
- Popularity of a particular thread (there are multiple threads per book, multiple
perspectives)
- Popularity of a specific tweet in the thread (#of comments, #of likes)
### User Interaction –
- Dedicated web-site, single page website, exactly like google homepage - User would enter – Book: Antifragile: Things That Gain from Disorder
- User would be able to see
    - Query box with the information(exactly like google)
    - Tags for the query
        - Type: book
        - Amazon Link
        - Good Reads link
        - Business & Economics 
        - Top 100 books of year
    - Book’s brief description
        - Books cover
        - Author
        - Main idea
    - Filters
        - Time Slider – since this is an important feature
        - Other advanced filters 
   - Sources
        - Author of the source
        - Link to the source 
        - Main idea
        - Time to read

<img width="400" alt="screen shot 2018-09-15 at 10 34 08 pm" src="https://user-images.githubusercontent.com/40564263/46065973-2fbaf380-c191-11e8-9264-4ceaf21a982f.png">

**Comment by p-raj** [comment]
## User Interaction - II

<img width="600" alt="screen shot 2018-09-26 at 1 39 12 pm" src="https://user-images.githubusercontent.com/40564263/46066199-bbcd1b00-c191-11e8-8d42-931141150557.png">

## Other Possibilities

<img width="600" alt="dezebriv4aaafii" src="https://user-images.githubusercontent.com/40564263/46066219-ca1b3700-c191-11e8-804d-887a5e9f9df6.jpg">

#### Some ideas are listed here - 
https://github.com/p-raj/v-fun/wiki/Ideas-%5C-JAN-%5C-Week-2-%5C-9th-14th-%5C-2017

**Comment by p-raj** [comment]
### Index Page - Version 0.0
![waiz](https://user-images.githubusercontent.com/40564263/46735771-d4225700-ccb4-11e8-97cb-60225cfb3f8d.png)

### Summary Page - Version 0.0
![waizer](https://user-images.githubusercontent.com/40564263/46735772-d4225700-ccb4-11e8-873f-e329973b9ab3.png)

### Link to the project  - Version 0.0 - https://github.com/p-raj/waiz
